src/binder/inc/clrprivbinderassemblyloadcontext.h

```diff
@@ -18,7 +18,32 @@ namespace BINDER_SPACE
     class AssemblyIdentityUTF8;
 };
 
-class CLRPrivBinderAssemblyLoadContext : public IUnknownCommon<ICLRPrivBinder>
+class AppDomain;
+
+#ifdef FEATURE_COLLECTIBLE_ALC
+
+class Object;
+class Assembly;
+class LoaderAllocator;
+
+class DECLSPEC_UUID("68220E65-3D3F-42E2-BAD6-2D07419DAB5E") IAssemblyLoadContext : public IUnknown
```

Comment:
I would suggest renaming the interface to be more incontext with its purpose - something like "ICollectibleAssemblyLoadContext".